aws-greengrass-testing-features/aws-greengrass-testing-features-api/src/main/java/com/aws/greengrass/testing/features/DeploymentSteps.java

```diff
@@ -245,7 +242,7 @@ private void executeCommand(int retryCount, List<String> commandArgs)
         try {
             String response = platform.commands().executeToString(CommandInput.builder()
                     .line(testContext.installRoot().resolve("bin").resolve("greengrass-cli").toString())
-                    .addAllArgs(commandArgs)
+                    .addArgs(formatToUnixPath(commandArgs))
```

Comment:
Why are we adding this? This already exists and this is why it cannot support json strings..